packages/zowe-explorer/src/globals.ts

```diff
@@ -54,6 +54,116 @@ export const ICON_STATE_OPEN = "open";
 export const ICON_STATE_CLOSED = "closed";
 export const VSCODE_APPNAME: string[] = ["Visual Studio Code", "VSCodium"];
 export const ROOTPATH = path.join(__dirname, "..", "..");
+export const DATA_SET_PROPERTIES = [
```

Comment:
Can you import the descriptions here: C:\Users\jp669971\Documents\GitHub\Community\vscode-extension-for-zowe\node_modules\@zowe\cli\lib\zosfiles\-strings-\en.d.ts

Reply:
And also, I think it needs to be localize for webpack

Reply:
How do I do this?

Reply:
Maybe we should talk to CLI team about it but they did it in this way : https://github.com/zowe/zowe-cli/blob/a9b40537e432935fda7b03e27467bb854cf3db2e/packages/cli/src/zosfiles/hMigrate/HMigrate.definition.ts#L18

Reply:
Or maybe @lauren-li and @JillieBeanSim can give us some hints on how to localize these properties

Reply:
I have done the localization but I am not sure how to import them into that file...so yeah it would be cool to ask someone about how to use that file :)

Reply:
The first part is right, I would like to import that file and use it. And I'm wondering if we could localize it without re-writing it in our codebase or can we use their own localization? (Maybe not 😢 )

Reply:
@jellypuno Assuming you mean that we should import descriptions from `@zowe\cli\lib\zosfiles-strings-\en.js` and display them localized in Zowe Explorer (as @JillieBeanSim described), I took a look at [this file](https://github.com/zowe/zowe-cli/blob/a9b40537e432935fda7b03e27467bb854cf3db2e/packages/cli/src/zosfiles/-strings-/en.ts ) in the Zowe CLI repo, and it seems we will need to do some JSON processing, especially since the keys there don't match the ones we need.

If we process the JSON and store their values in variables in Zowe Explorer, then I believe we should just be able to localize the values of those variables per our usual localization method.

Reply:
so we would need to create a schema that could be used to process the JSON and could set a way to localize all Description fields?

Reply:
That might be one way to do it, although perhaps in a separate PR.

Reply:
I agree. Thank you for your guidance Lauren and Billie. I appreciate it 😊

@katelynienaber Just use the file for now and then the localization will be handled on a different PR. 😉

Reply:
I just had the thought that if we are using the imported strings file from `@zowe`, we'll be able to pull in the latest updated strings, but that means we probably won't be able to localize the strings ourselves. 🤔 But perhaps we could set some variable to update the path to the import based on the user's VS Code language setting, and use whatever localization is provided by `@zowe` (although currently, this only appears to be English).